frontend/src/components/DevicesBrowser.vue

```diff
@@ -7,16 +7,8 @@
             v-if="loading"
             message="Loading Devices..."
         />
-        <ff-loading
```

Comment:
Why remove these loading indicators?

Reply:
> Right now, the loading spinners replace the entire page, meaning it flashes and then the entire page is re-rendered, resetting the state
> In a previous PR I changed the popovers to only show the toast popup after the request completes
> And then the table updates it’s contents reactively, instead of doing a full re-render